### PR TITLE
add sus board

### PR DESCRIPTION
**Pull request by alabahuy**
## Description

add sus board, macropad 3x4

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/ocean/sus/config.h [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+/* COL2ROW, ROW2COL*/
+#define DIODE_DIRECTION COL2ROW
+
+/* Debounce reduces chatter (unintended double-presses) - set 0 if debouncing is not needed */
+#define DEBOUNCE 5
+
+/* Mechanical locking support. Use KC_LCAP, KC_LNUM or KC_LSCR instead in keymap */
+#define LOCKING_SUPPORT_ENABLE
+/* Locking resynchronize hack */
+#define LOCKING_RESYNC_ENABLE
```
Comment (drashna) [inline keyboards/ocean/sus/config.h:55]: These really aren't needed, as I don't think anyone has locking switches. 
```suggestion
```

File: keyboards/ocean/sus/readme.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+![SUS](https://user-images.githubusercontent.com/30220306/126883340-47401d95-2095-4e55-ba5c-4ae7acaa430f.png)
+
+
+A Macropad like a amogus
+
+* Keyboard Maintainer: Ocean
+* Hardware Supported: Sus Board, Pro Micro
+* Hardware Availability: Private GB
+
+Make example for this keyboard (after setting up your build environment):
```
Comment (drashna) [inline keyboards/ocean/sus/readme.md:12]: Could you add instructions on how to put the keyboard into bootloader mode, to flash new firmware?

File: keyboards/ocean/sus/rules.mk [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+MCU = atmega32u4
+
+# Bootloader selection
+#   Teensy       halfkay
+#   Pro Micro    caterina
+#   Atmel DFU    atmel-dfu
+#   LUFA DFU     lufa-dfu
+#   QMK DFU      qmk-dfu
+#   ATmega32A    bootloadHID
+#   ATmega328P   USBasp
```
Comment (drashna) [inline keyboards/ocean/sus/rules.mk:11]:
```suggestion
```

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/ocean/sus/config.h (hunk 1 above)
Comment (alabahuy) [reply keyboards/ocean/sus/config.h]: @drashna oh i see, thanks you

File: keyboards/ocean/sus/readme.md (hunk 2 above)
Comment (alabahuy) [reply keyboards/ocean/sus/readme.md:12]: @drashna done :)

**Review by drashna — APPROVED** [review]

File: keyboards/ocean/sus/rules.mk [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = no         # Console for debug
+COMMAND_ENABLE = no         # Commands for debug and configuration
+NKRO_ENABLE = yes           # Nkey Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+MIDI_ENABLE = no            # MIDI controls
+AUDIO_ENABLE = no           # Audio output
+UNICODE_ENABLE = no         # Unicode
+BLUETOOTH_ENABLE = no       # Enable Bluetooth
+RGBLIGHT_ENABLE = no        # Enable WS2812 RGB underlight.
```
Comment (fauxpark) [inline keyboards/ocean/sus/rules.mk:22]:
```suggestion
#   change yes to no to disable
#
BOOTMAGIC_ENABLE = no       # Virtual DIP switch configuration
MOUSEKEY_ENABLE = yes       # Mouse keys
EXTRAKEY_ENABLE = yes       # Audio control and System control
CONSOLE_ENABLE = no         # Console for debug
COMMAND_ENABLE = no         # Commands for debug and configuration
# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
NKRO_ENABLE = yes           # USB Nkey Rollover
BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
BLUETOOTH_ENABLE = no       # Enable Bluetooth
AUDIO_ENABLE = no           # Audio output
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/ocean/sus/rules.mk (hunk 4 above)
Comment (alabahuy) [reply keyboards/ocean/sus/rules.mk]: done :) @fauxpark

**Review by fauxpark — APPROVED** [review]

**Comment by alabahuy** [comment]
any update ? @drashna @fauxpark

File: keyboards/ocean/sus/config.h [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+ *                  ROW2COL = ROW = Anode (+), COL = Cathode (-, marked on diode)
+ *
+ */
+/*                        0    1    2    3    4    5    6    7    8    9   10  11  12  13  14*/
+#define MATRIX_ROW_PINS { B5, B4, E6, D7 }
+#define MATRIX_COL_PINS { C6, D4, D0 }
+#define UNUSED_PINS
+
+/* COL2ROW, ROW2COL*/
+#define DIODE_DIRECTION COL2ROW
```
Comment (obosob) [inline keyboards/ocean/sus/config.h:47]: looking at the board it looks like the diode footprints are backwards (see below). checked a [video build guide](https://www.youtube.com/watch?v=bYs-EQz-5dY) too to see if they reccommend reversing the diode but you can see they also do solder the diodes with the cathode connected to the switch and the anode to the row trace. this orientation won't work with row2col but *should* work, albeit non-conventionally, with row2col. Has this exact firmware been tested on the board?

image

**Review by obosob — CHANGES_REQUESTED** [review]

File: keyboards/ocean/sus/config.h (hunk 5 above)
Comment (alabahuy) [reply keyboards/ocean/sus/config.h:47]: @obosob in the picture there is an error in the polarity of the diode, and this board uses row2col. thank you very much for your attention.  image

**Comment by drashna** [comment]
Thanks!

**Pull request merged**